test/test_check.py

```diff
@@ -7,11 +7,10 @@
 from asv import util
 
 from . import tools
-from .test_dev import basic_conf
 
 
 def test_check(capsys, basic_conf):
-    tmpdir, local, conf = basic_conf
+    tmpdir, local, conf, _ = basic_conf
```

Comment:
What's the reasoning here? Why do you think this is the best solution?

Reply:
Could have used * operator to pack the remaining elements into a list but  since it was a single element remaining to be returned i decided to use _ the dummy variable to unpack it (as what was expected was 3 and the values returned on the left were 4 , the remaining element had no variable, i assigned a dummy variable _)

Reply:
Did you check why the function returns 4, and 3 are expected? My question is not about why underscore or star. The question is why if this used to return 3, why it now returns 4, and receiving one more parameter is your preferred solution (as opposed to make the function to keep returning 3, as it was before)?

Reply:
Yes,An extra variable machine_file was added to be returned that stores the hostname machine details. This makes the variables to be returned 4, and yet when the function was being called in different files earlier it was unpacking 3 variables, hence the need to add the dummy variable to match what the function is returning.

Reply:
I see. But this don't make sense. The reason why a different number of variables are being returned is because with your changes, a different function is being called. So different things are happening in the tests in your branch, and in master. Which is obviously something we don't want, unless you've got a reason to change the behavior, which doesn't seem it is the case.